templates/projects/splash.macros.j2

```diff
@@ -0,0 +1,48 @@
+{% import 'utility.macros.j2' as utility %}
+{% import 'birds/utility.macros.j2' as birds %}
+
+
+{% macro splash_list(has='client', slugs=none) %}
+  <section class='splash-projects extend-full'>
+    {% if slugs %}
+      {% for slug in slugs %}
+        {% set page = all_pages|filter_pages('slug', 'eq', slug)|get_page %}
+        {{ splash(page) }}
+      {% endfor %}
+    {% else %}
+      {% set pages = all_pages|filter_pages(has)|filter_pages('public', 'neq', false) %}
+
+        {% for page in pages %}
+          {{ splash(page) }}
+        {% endfor %}
+    {% endif %}
+  </section>
+{% endmacro %}
+
+
+{% macro splash(page) %}
+  {% set project = page.config.project[0] %}
+  {% set client = page.config.client[0] if page.config.client else none %}
+  {% set contrib = page.config.contributors[0] if page.config.contributors else none %}
+
+  {% set link = ['/', page.slug, '/']|join('') if (project.owner or project.case_study) else '/work/' %}
```

Comment:
It seems a little odd to me that we have pages for open source work whether we created it or not (e.g. `/open-source/django/`, `/open-source/true/`), but we only link to the standalone page if we created it? Minor point; I just don't understand the intention behind the structure.